From adc1b71d84ab8b618c623749f43dc6350d75de6b Mon Sep 17 00:00:00 2001
From: Michal Sekletar <msekleta@redhat.com>
Date: Tue, 4 Mar 2014 17:00:54 +0100
Subject: [PATCH] logind: given that we can now relatively safely shutdown
 sessions copes without working cgroup empty notifications there's no need to
 set the stop timeout of sessions scopes low

Based-on: a50df72b37ce2a7caf7775c70d18c3f9504b9e80
---
 src/login/logind-dbus.c | 11 +----------
 1 file changed, 1 insertion(+), 10 deletions(-)

diff --git a/src/login/logind-dbus.c b/src/login/logind-dbus.c
index a68e9b4..6cabb7b 100644
--- a/src/login/logind-dbus.c
+++ b/src/login/logind-dbus.c
@@ -2549,10 +2549,9 @@ int manager_start_scope(
                 DBusError *error,
                 char **job) {
 
-        const char *timeout_stop_property = "TimeoutStopUSec", *send_sighup_property = "SendSIGHUP", *pids_property = "PIDs", *after_property = "After";
+        const char *send_sighup_property = "SendSIGHUP", *pids_property = "PIDs", *after_property = "After";
         _cleanup_dbus_message_unref_ DBusMessage *m = NULL, *reply = NULL;
         DBusMessageIter iter, sub, sub2, sub3, sub4;
-        uint64_t timeout = 500 * USEC_PER_MSEC;
         dbus_bool_t send_sighup = true;
         const char *fail = "fail";
         uint32_t u;
@@ -2632,14 +2631,6 @@ int manager_start_scope(
          * stop timeout for sessions, so that we don't wait
          * forever. */
 
-        if (!dbus_message_iter_open_container(&sub, DBUS_TYPE_STRUCT, NULL, &sub2) ||
-            !dbus_message_iter_append_basic(&sub2, DBUS_TYPE_STRING, &timeout_stop_property) ||
-            !dbus_message_iter_open_container(&sub2, DBUS_TYPE_VARIANT, "t", &sub3) ||
-            !dbus_message_iter_append_basic(&sub3, DBUS_TYPE_UINT64, &timeout) ||
-            !dbus_message_iter_close_container(&sub2, &sub3) ||
-            !dbus_message_iter_close_container(&sub, &sub2))
-                return log_oom();
-
         /* Make sure that the session shells are terminated with
          * SIGHUP since bash and friends tend to ignore SIGTERM */
         if (!dbus_message_iter_open_container(&sub, DBUS_TYPE_STRUCT, NULL, &sub2) ||
